### PR TITLE
Astra Assistants Support

### PR DESCRIPTION
**Pull request by phact**
Added a few components to support [astra-assistants](https://github.com/datastax/astra-assistants-api/tree/main), an open source implementation of OpenAI's assistants API that supports different LLM and embedding providers.

### CODE REVIEW

**Comment by ogabrielluiz** [comment]
Hey @phact !! 

How are you? 

This looks so cool!

**Comment by phact** [comment]
Hey @ogabrielluiz , doing great :) 

Excited about this first stab at Assistants support. This is my first component contribution. What steps do you recommend to get it ready to merge? I plan to continue improving and maintaining Assistants support in langflow going forward if that works for you and he team.

File: src/backend/base/langflow/config.yaml
Comment (ogabrielluiz) [inline src/backend/base/langflow/config.yaml:1]: You don't need this. We are removing this config.

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @phact 

You might want to add `build_config` to the components so the Fields have better names and information on how to use them.

**Comment by phact** [comment]
Hi @ogabrielluiz any additional feedback now that your suggestions were addressed?

Thanks!!!

**Comment by phact** [comment]
Hey @ogabrielluiz just checking in. Congrats on the v1 launch!

**Comment by phact** [comment]
@ogabrielluiz checking in on this

**Comment by ogabrielluiz** [comment]
Thank you, Seb! We'll take a stance on this PR this week. I'll change the base to `main` and pick a category for them.

**Review by ogabrielluiz — APPROVED** [review]

**Pull request merged**